info-commands.go

```diff
@@ -331,9 +331,9 @@ type ServerProperties struct {
 
 // DiskMetrics has the information about XL Storage APIs
 // the number of calls of each API and the moving average of
-// the duration of each API.
+// the duration, in nanosecond, of each API.
 type DiskMetrics struct {
-	APILatencies map[string]string `json:"apiLatencies,omitempty"`
+	APILatencies map[string]uint64 `json:"apiLatencies,omitempty"`
```

Comment:
```suggestion
	APILatencies map[string]interface{} `json:"apiLatencies,omitempty"`
```

And convert it in `mc` @vadmeste

Reply:
> And convert it in `mc` @vadmeste

yes, good idea!